routes/article.js

```diff
@@ -112,35 +120,32 @@ router.post('/vote', (req, res) => {
                 word = word.toLowerCase()
                 console.log(word)
                 Keyword.find({word: word}).then((found) => {
```

Comment:
Same here (about the naming of found)

Use `findOne` since it looks like you only want one anyways